WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -355,9 +356,21 @@ public async Task<Coin> OutpointToCoinAsync(InputRegistrationRequest request, Ca
 	{
 		OutPoint input = request.Input;
 
-		if (Prison.TryGet(input, out var inmate) && (!Config.AllowNotedInputRegistration || inmate.Punishment != Punishment.Noted))
+		if (Prison.TryGet(input, out var inmate))
 		{
-			throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.InputBanned);
+			var bannedUntil = inmate.Punishment == Punishment.LongBanned ? inmate.Started + Config.ReleaseUtxoFromPrisonAfterLongBan : inmate.Started + Config.ReleaseUtxoFromPrisonAfter;
```

Comment:
Warden is handling the release of the inmates, you do not have to calculate anything here.

Reply:
I don't get this. Warden won't tell me the remaining time of the ban and I don't want to release anyone here.